substrate/client/network/types/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+description = "Substrate network types"
+name = "sc-network-types"
+version = "0.10.0-dev"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+authors.workspace = true
+edition.workspace = true
+homepage = "https://substrate.io"
+repository.workspace = true
+documentation = "https://docs.rs/sc-network-types"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[build-dependencies]
+prost-build = "0.11"
+
+[dependencies]
+bs58 = "0.4.0"
+libp2p-identity = { version = "0.1.3", features = ["peerid"] }
+multiaddr = "0.17.0"
+multihash = { version = "0.17.0", default-features = false, features = ["std", "multihash-impl", "identity", "sha2"] }
+rand = "0.8.5"
+thiserror = "1.0.48"
+
+[dev-dependencies]
+tempfile = "3.1.0"
```

Comment:
Looks like not all of these deps are necessary? (At least for the very first commit, which is what I'm looking at right now.)